portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/OpenIDConnect.testcase

```diff
@@ -698,6 +821,290 @@ definition {
+			Navigator.openURL(baseURL = "http://www.able.com:8080");
+
+			AssertClick(
+				locator1 = "UserBar#USER_SIGN_IN",
+				value1 = "Sign In");
+
+			Click(locator1 = "Icon#OPENID_CONNECT");
+
+			Click(locator1 = "Button#SIGN_IN");
+
+			GoogleLogin.acceptConditions();
+
+			MyAccount.openMyAccountAdmin(baseURL = "http://www.able.com:8080");
+
+			User.viewUserInfomation(
+				userBirthday = "12/12/1990",
+				userEmailAddress = "tester1@liferay.com",
+				userFirstName = "First1",
+				userLastName = "Last1",
+				userMiddleName = "Middle Name",
+				userScreenName = "user1");
+		}
+	}
+
+	@description = "This is a use case for LPS-162080 TC-3: User information should be synced towards Keycloak too. Verify that User role can be imported to an existing User on Keycloak from Portal."
+	@priority = "5"
+	test UserInformationsCanBeSyncedToKeycloak {
+		property custom.properties = "open.id.connect.user.info.processor.impl.issuer=http://127.0.0.1:18080/auth/realms/master${line.separator}open.id.connect.user.info.processor.impl.regular.role=Power User";
+		property osgi.module.configuration.file.names = "com.liferay.portal.security.sso.openid.connect.configuration.OpenIdConnectConfiguration.config";
+		property osgi.module.configurations = "enabled=\"true\"";
+		property test.name.skip.portal.instance = "OpenIDConnect#UserInformationsCanBeSyncedToKeycloak";
+
+		task ("Given: User starts Keycloak for test and set debug level") {
+			AntCommand(
+				locator1 = "build-test-keycloak.xml",
+				value1 = "prepare-selenium-keycloak-server");
+
+			OpenIDConnect.enableOpenIDConnectDebugLogs();
+
+			OpenIDConnect.enableOpenIDConnectCustomParametersDebugLogs();
+		}
+
+		task ("And: Add Auth Server Local Metadata") {
+			OAuthClient.openOAuthClientAdmin(navbar = "Auth Server Local Metadata");
+
+			OAuthClient.addAuthServer(jsonFileName = "keycloak-local-metadata.json");
+
+			var wellknownURI = OAuthClient.getWellKnownURI();
+		}
+
+		task ("And: Add OAuth Client") {
+			OAuthClient.openOAuthClientAdmin(navbar = "Clients");
+
+			OAuthClient.addClient(
+				jsonFileName = "keycloak.json",
+				wellknownURI = "${wellknownURI}");
+		}
+
+		task ("When: User login as keycloak user") {
+			User.logoutPG();
+
+			User.loginUserWithOpenIDConnectEnabledPG(
+				userEmailAddress = "user2",
+				userPassword = "test");
+
+			User.viewLoggedInPG();
+		}
+
+		task ("Then: User information is synced on both sides, User has Power User role on Keycloak") {
+			var userAccessToken = OpenIDConnect._getKeycloakUserAccessToken(userEmail = "tester1@liferay.com");
+
+			var userInformation = OpenIDConnect._getKeycloakUserInformations(accessToken = "${userAccessToken}");
+
+			if (!(contains("${userInformation}", "Power User"))) {
+				fail("Keycloak didn't import user role.");
+			}
+		}
+	}
+
+	@description = "This is a use case for LPS-162080 TC-1: OpenID configuration mandatory fields should work with unique error messages. Blocked by LPS-164921."
+	@ignore = "true"
```

Comment:
Since LPS-164921 is a minor bug, it's no need to ignore the whole test case. We could comment on the validation for the LPS-164921 part. Then we will not miss any other important validation.

Reply:
The bug occurring based on the bug ticket will be present after every second save. 
In order for this test to work, I would need to exit the edit and start it over again in order not to encounter this issue

Reply:
Thanks for the info. I raised LPS-164921 to FP3 according to this situation.